tools/WinObjC.Language/WinObjC.Language.Packaging.targets

```diff
@@ -3,11 +3,12 @@
 
   <Target Name="AddBuiltOutput" BeforeTargets="GetPackageContents" Condition="'$(SkipGetPackageContentsForOtherPlatforms)' != 'true'">
       <ItemGroup>
-          <!-- Inlcude *all* of /include/  and /include_next/ folder.  -->
-          <_IncludeFiles Include="$(MSBuildThisFileDirectory)..\include\**\*.*"/>
+          <!-- Include *all* of /include/, /include/xplat/ and /include_next/ folder.  -->
+          <_IncludeFiles Include="$(MSBuildThisFileDirectory)..\include\**\*.*" Exclude="$(MSBuildThisFileDirectory)..\include\xplat\**\*.*"/>
+          <_IncludeFiles Include="$(MSBuildThisFileDirectory)..\include\xplat\**\*.*"/>
```

Comment:
I'm conflicted because that seems like the more correct thing, but also would involve touching like 80 different files. At that point, the "add a SmartTypes.h that's just \#include \\<..\..\..\tools\include\Starboard\SmartTypes.h\> to where it used to be" solution starts seeming preferable.